frontend/components/Main/MainHeader/index.tsx

```diff
@@ -274,27 +274,20 @@ export const MainHeader = () => {
         totalEthBalance > requiredGas
       );
     })();
-
     const serviceExists = !!services?.[0];
 
-    if (!isDeployable) {
-      return (
-        <Button type="default" size="large" disabled>
-          Start agent {!serviceExists && '& stake'}
-        </Button>
-      );
-    }
+    const canActivateAgent = canStartAgent && (isAgentEvicted || isDeployable);
```

Comment:
getting lost with these `can` variables, they need more context

i.e. `canStartAgent` is not solely determined by the staking contract, and needs to be renamed to something more fitting, unsure what exactly but it should infer it's relation to the staking contract

`canActivateAgent`, i'm lost on what this represents too :/ canStartAgent already factors in isAgentEvicted

![image](https://github.com/user-attachments/assets/bf400f16-9ef5-48f6-93c8-7707b4bacc54)